notebooks/curation_api/R/get_collection_version_R.ipynb

```diff
@@ -0,0 +1,175 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5490d93",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Fetch a Collection Version with full Dataset metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "The script in this notebook fetches full Collection Version metadata for a given Collection Version.\n",
+    "\n",
+    "Fetching an individual Collection Version requires only the Collection Version id, and not an API key/access token."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "edfb1272",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1cff472c",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "library(\"readr\")\n",
+    "library(\"httr\")\n",
+    "library(\"stringr\")\n",
+    "library(\"rjson\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97459362",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "#### <font color='#bc00b0'>Please fill in the required values:</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb7eb2ae",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter the id of the Collection Version you want to fetch</font>\n",
+    "\n",
+    "_A Collection Version id can be found by making a call to the `/collections/{collection_id}/versions` endpoint for a particular Collection, using its canonical collection ID._"
```

Comment:
@nayib-jose-gloria this should point to `GET /collections/{collection_id}` instead